.github/pull_request_template.md

```diff
@@ -1,39 +1,33 @@
 ## Description
+<!--- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
 
-A detailed explanation of the changes in your PR.
-
-Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.
 
 ## Checklist
-- [ ] Investigated and inspected CI test results
-- [ ] Unit test and regression tests added
-- [ ] Evaluated and added CHANGELOG entry if required
-- [ ] Determined and documented upgrade steps
-- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))
-
-If any of these don't apply, please comment below.
+All items *must be* checked:
+- [ ] CI results are inspected
+- [ ] CHANGELOG is updated **OR** not needed
+- [ ] Documentation PR is created and linked above **OR** not needed
 
-## Testing Performed
+## Testing
+- [ ] added unit, e2e, regression, and/or compatibility tests
+- [ ] modified existing tests
+- [x] **no new tests added**
+  <!--- Please explain why unless it's obvious, e.g., the PR is a one-line comment change. -->
 
-### Here I tell how I validated my change
+### How I validated my change
 
-TODO(replace-me)  
+<!---
 Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
 Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
 provided convince any reviewer or any external reader that you did enough to validate your change.
 
 It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.
 
-It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
+It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.
 It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
 responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
-ASAP after it gets merged or explain in PR when the validation will be performed.  
+ASAP after it gets merged or explain in PR when the validation will be performed.
 Explain here why you skipped testing in case you did so.
 
 Have you created automated tests for your change? Explain here which validation activities you did manually and why so.
-
-### Reminder for reviewers
-
-In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
-performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
-performed testing and you are satisfied with it.
```

Comment:
I'd keep this. Could be shortened to:

```suggestion
### Reminder for reviewers

As a reviewer, I confirm that I reviewed testing performed by the author and that, by approving this change, I share responsibility with the author for any problems this code may cause.
```

Reply:
Do you think having this statement will increase the quality or reviewers' engagement? While I do share the sentiment about sharing the responsibility, the way you suggest to formulate it sounds like a clause in some bank contract which is repelling.

Reply:
If reviewers understand and share the idea, there's no need to write anything. If you agree that reviewers are all over the place, it would make sense to have something short but to the point in order to help them recognize why they do this.

Reply:
I think a reminder would be good but given our pool of reviewers is rather static, I'm pretty sure their brains learn quickly to disregard static and annoying parts of the PR, like ads on some website. My goal here is to remove as much boilerplate as possible, so the reviewer's brain is constantly stimulated with varying combinations of checked items.

I'd be willing to try this approach, we can always (re-)add an explicit reminder if we'll see it worked better that way

Reply:
Ok, let's not bother now.

Reply:
🙏